### DISCUSSION TITLE
IROS competition starts today!

### DISCUSSION THREAD

**Discussion by omichel** [cyberbotics/wrestling#98] [Announcements]
# 🎉 The new IROS competition is starting today! 🎉
We took into account your [feedback](https://github.com/cyberbotics/wrestling/discussions/92) to improve the competition for the IROS 2023 edition.
The home page of the new competition is still [here](https://t.co/Y8mdl9NsI3). As you can see, we removed all the entries from the leader board (except the examples which were updated). We recommend that you create a new repository from [this one](https://github.com/cyberbotics/wrestling) as quite a few [things changed](https://github.com/cyberbotics/wrestling/pull/96) (world file, proto files, referee, etc.). Making a single commit in your new repository will allow you to enter the new competition. However, you should be able to port your previous controller directly from your ICRA repository to the new one as nothing changed in the API and their behavior with the new scenario should be pretty the same as with the previous one.

## Good luck for the new IROS competition!

*note*: Please do not commit anything to your old ICRA repository as it would create a duplicate entry for you, which is forbidden. It is recommended to delete your ICRA repository once you moved your controller files to your new IROS repository.

**Comment by omichel** [comment]
The new competition relies on Webots R2023b which we are going to officially release very soon. Meanwhile, we recommend that you use a [nightly build of Webots R2023b](https://github.com/cyberbotics/webots/releases/) to develop and test your robot controllers.

**Reply by omichel** [reply]
We are currently having issues with the using the new version of Webots R2023b on our simulation server. We are working on resolving the issue on #99. I will keep you posted here. Once resolved the online games and leader board should work as before.

**Reply by omichel** [reply]
I believe the problems are now fixed and the competition is running. Please report here if you find any other problem.

**Comment by omichel** [comment]
The competition is now listed on the [official IROS website](https://ieee-iros.org/competitions).

**Comment by omichel** [comment]
We just introduced a new feature in the competition: the ability to run friendly test games versus the opponent of your choice. The outcome of such a game does not affect the ranking and the corresponding 3D animation is visible from the [leader board](https://webots.cloud/run?version=R2023b&url=https://github.com/cyberbotics/wrestling/blob/main/worlds/wrestling.wbt&type=competition) under the new `Test` button. If you already created your repository from the [template](https://github.com/cyberbotics/wrestling), you may have to add this new [action script](https://github.com/cyberbotics/wrestling/blob/main/.github/workflows/test.yml) to your repository. Then, simply follow the [instructions](https://github.com/cyberbotics/wrestling#friendly-test-games) to trigger the friendly test game of your choice.

**Reply by MJavadZallaghi** [reply]
Interesting feature!
Thanks for creating this one.

**Comment by amirhnourian** [comment]
Dear @omichel:
Hi; 

I pushed the latest changes to my robot but unfortunately, it is not competing against the Demo! 
Can you please check this? Am I doing something wrong here?

Thanks

**Reply by omichel** [reply]
It could be that the runner machine was rebooting to make some update. I just restarted an evaluation of your controller and it seems it worked, see https://github.com/cyberbotics/wrestling/actions/runs/5783190379/job/15671501420
